lib/Data/Aeson/Combinators/Compat.hs

```diff
@@ -0,0 +1,67 @@
+{-# LANGUAGE CPP #-}
+-- |
+-- Module      : Data.Aeson.Combinators.Decode
+-- Copyright   : (c) Marek Fajkus
+-- License     : BSD3
+--
+-- Maintainer  : marek.faj@gmail.com
```

Comment:
I volunteered you as the maintainer. Hope that's OK, but I am also fine with my name and email here if you'd prefer that